### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by ParkerPattillo** [TobiasPankner/Teams-Auto-Joiner#117] [General]
I just wanted to come here at the end of my senior year of high school to thank you for taking the time to create this code and making available to people like me. This made the online school classes I had to join easy and most definitely helped me get extra hours of sleep. I would have definitely been counted absent for online classes numerous times if it hadn't been for this wonderful code. Your constant updates and improvements to the code were very helpful as well. With the code I was perfectly able to automate my online schooling experience. I'm in your debt dude, thank you!

**Comment by TobiasPankner** [comment]
You're welcome!  
It's great to know that my code was helpful to you.

**Comment by diegorojasc88** [comment]
Really useful tool, thanks a lot!
